cmd/poold/main.go

```diff
@@ -18,7 +18,7 @@ var (
 func main() {
 	err := start()
 	if err != nil {
-		fmt.Println(err)
+		_, _ = fmt.Fprintf(os.Stderr, "[poold] %v\n", err)
 		os.Exit(1)
```

Comment:
Consider the following so we don't have to specify a log format literal?
```
log.SetOutput(os.Stderr)
log.Fatalf("startup error: %v", err)
```

(`log.Fatalf` will call `os.Exit(1)`)

I'm not sure where the log formatted is initiated.

Reply:
We don't really use the standard library logger anywhere. And if we did that here, then it would add its own header (date/time, file and line number) and stack trace. And the date format will be in another format, so setting that up would be additional code. We just want to make sure things end up in `stderr`, which we do here as well: https://github.com/lightningnetwork/lnd/blob/f9f8f6406b110168553d4e4b6aae7a857ef9fe3c/config.go#L800